hasjob/models/domain.py

```diff
@@ -104,6 +104,11 @@ def get(cls, name, create=False):
             db.session.add(result)
         return result
 
+    @classmethod
+    def autocomplete(cls, prefix):
+        search = prefix + '%'
```

Comment:
Prefix will need to be sanitized to escape `%` and other characters. https://github.com/hasgeek/lastuser/blob/master/lastuser_core/models/user.py#L338-L340

Reply:
This may be generic enough to be a coaster util.